radio/src/targets/flysky/AFHDS2A_a7105.cpp

```diff
@@ -22,6 +22,18 @@
 #define AFHDS2A_HUB_TELEMETRY
 //#define AFHDS2A_NUMFREQ			16
 
+#ifdef AFHDS2A_CHANNELS 
+# ifdef AFHDS2A_LQI_CH
+#  undef AFHDS2A_LQI_CH
```

Comment:
Could this be used to inject LQI at ch17 like in multimodule?

Reply:
If FlySky is doing it this way then i'm completely fine with your approach. Is Multimodule also going this way?

Reply:
But we are clamped at 16channels on FS-i6X, so those last two are of no use, so it might be a good idea to keep LQI at ch17.

Reply:
Oh yes, just had a look into `dataconstants.h` ! Wasn't aware of this fact.

In this case, I totally agree with. I'll change that.

Reply:
Increasing the `MAX_OUTPUT_CHANNELS` to 18 gets me 124 Bytes of free ram instead of 172 free byte wir 16 channels.

Is this without the stack?

I don not need CROSSFIRE or ELRS or telemetry output. Is it worth to disable these to get more free RAM?

Reply:
Non standard values (increase by 8) need additional logic in few places - channels pagination and joystick support, so I think that it must be 16 or 24. I would like to keep it at 16 and you may use 24 for your builds (maybe with additional logic to do not read channelOutputs above avail channels).

Reply:
Another potential issue: `AFHDS2A_TXPACKET_SIZE` should be 38 instead of 37 (accordimg to MPM code).
And: if I use `uint8_t packet[AFHDS2A_TXPACKET_SIZE]` with 37 I get an index out of bounds error.